src/web_app/js/peerconnectionclient.js

```diff
@@ -174,9 +174,9 @@ PeerConnectionClient.prototype.getPeerConnectionStats = function(callback) {
 
 PeerConnectionClient.prototype.doAnswer_ = function() {
   trace('Sending answer to peer.');
-  this.pc_.createAnswer(this.setLocalSdpAndNotify_.bind(this),
-      this.onError_.bind(this, 'createAnswer'),
-      PeerConnectionClient.DEFAULT_SDP_CONSTRAINTS_);
+  this.pc_.createAnswer()
```

Comment:
The answer constraints should not be removed, webrtc looks for it to decide if it receives media or uses bundle:

https://code.google.com/p/chromium/codesearch#chromium/src/third_party/libjingle/source/talk/app/webrtc/mediastreamsignaling.cc&rcl=1437050479&l=442